forge-gui/res/cardsfolder/upcoming/vulpine_harvester.txt

```diff
@@ -0,0 +1,11 @@
+Name:Vulpine Harvester
+ManaCost:3 W
+Types:Creature Phyrexian Fox
+PT:3/3
+T:Mode$ AttackersDeclared | ValidAttackers$ Phyrexian.YouCtrl | Execute$ TrigReturn | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more Phyrexians you control attack, return target artifact card from your graveyard to the battlefield if its mana value is less than or equal to their total power.
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Artifact.YouCtrl+cmcLEX | TgtPrompt$ Select target artifact in your graveyard | Origin$ Graveyard | Destination$ Battlefield
```

Comment:
![image](https://user-images.githubusercontent.com/8506892/230713086-46c26176-55e1-4983-8118-8a5f216b43cb.png)

Reply:
So a ChooseCard variable with a ConditionDefined$ Targeted or is there another way ?

Reply:
Try `ValidTgts$ Artifact.YouCtrl | ConditionDefined$ Targeted | ConditionPresent$ Card.cmcLEX`